garden/src/features/entrypoints/components/AssociatedMaterialCards.tsx

```diff
@@ -19,17 +19,44 @@ export const PaperCard = ({
   onUpdate: (d: Paper) => void;
   onDelete: (index: number) => void;
 }) => {
+  const paperLink = paper.url || (paper.doi ? `https://doi.org/${paper.doi}` : undefined);
+
   return (
     <Card className="transition-colors hover:bg-gray-50 hover:shadow-lg">
       <CardHeader>
         <CardTitle className="text-xl font-bold text-gray-900 transition-colors duration-300">
-          {paper.title}
+          {paperLink ? (
+            <a 
+              href={paperLink} 
+              target="_blank" 
+              rel="noopener noreferrer"
+              className="hover:text-blue-600 hover:underline"
+            >
+              {paper.title}
+            </a>
+          ) : (
+            paper.title
+          )}
         </CardTitle>
         {paper.doi && (
           <div className="flex space-x-3">
             <span>DOI: {paper.doi}</span>
           </div>
         )}
+        {paper.url && (
+          <div className="flex space-x-3 mt-1">
+            <span className="text-sm">
+              URL: <a 
+                href={paper.url} 
```

Comment:
What's up with having both a paperLink and paper.url? Are those distinct?

Reply:
yeah there should just be one of these, good catch